07_cmd_lab/cmdlab.txt

```diff
@@ -1,13 +1,23 @@
-Intro To Command Line Lab.
+Intro To Command Line Lab - Amir Meiron
 Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
+  echo $0 > bash
   - your default shell?
+  echo $shell > /bin/bash
   - a list of all shells currently installed on the machine?
-
-2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
+  cat /etc/shells:
+	/bin/sh
+	/bin/dash
+	/bin/bash 
+	/bin/rbash
+2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". 
+What information is held in the file? - those are all the users in the system with their attributies.
+What are the fields in each line? login name : optional encrypted password : numerical user ID : numerical group ID : user name or comment field : user home directory : optional user command interpreter.
 
 3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+cd \etc > ls -S -f
```

Comment:
right... but you *DO* want to sort the files don't you? :) 
so how would you fix it?